src/App.svelte

```diff
@@ -10,12 +10,21 @@
 	import Templates from './pages/Templates.svelte';
 
     export let url = '';
-    
+
     let userdata = {};
 
     onMount(async () => userdata = await userData());
     const toggleusermenu = () => jQuery(".usermenu").slideToggle();
 
+	// Load Drip picture as fallback if avatar is not available
+    const loaddrip = () => {
+		if (userdata.drip) {
+			useravatar.src = userdata.drip;
+		} else {
+			useravatar.src = "/css/img/nodrip.png";
```

Comment:
For consistency, it might be better to use /css/img/discord.png instead of /css/img/nodrip.png.  Reason being, if the user doesn't have a profile picture set, discord.png is the default.  I suppose it would be neat to use the drip pic, I could just see scaling issues if the user put in a low resolution image.